src/models/Pet.js

```diff
@@ -5,70 +5,69 @@ const PetSchema = new mongoose.Schema(
     petName: {
       type: String,
       required: [true, 'Please provide pet name'],
-      maxlength: [100, 'Name can not be more than 100 characters'],
+      //maxlength: [100, 'Name can not be more than 100 characters'],
     },
     petType: {
       type: String,
-      enum: ['Cat', 'Dog', 'Other'],
-      default: 'Cat',
-      required: [true, 'Please provide type'],
+      //required: [true, 'Please provide pet type'],
```

Comment:
Any specific reason for removing the enums from the model? Was just wondering.

Reply:
We think that enums were interfering the proper work of fetching file